### DISCUSSION TITLE
Is it possible to disable the high pitched "ding" before a TTS announcement?

### DISCUSSION THREAD

**Discussion by addrum** [music-assistant/support#913] [Q&A] [answered]
I've noticed when I send a TTS to a Google Cast speak there's a high pitched "ding" before the message is played; is there any way to disable this? Apologies I couldn't find anything on this already so please point me in the right direction if it's documented somewhere.

**Comment by OzGav** [comment, marked as answer]
https://github.com/music-assistant/support/discussions/896
